index.html

```diff
@@ -1,12 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>TestB</title>
-</head>
-<body>
-    <h1>Hello Test B</h1>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <h1 id="title">Yeusof & Geeky Trail</h1>
```

Comment:
It would be better to include the h1 and the following paragraph inside a header tag as this will make your page more accessible and better semantically.

Reply:
- [ ] Good job you did here 👏 . However, when writing out tags, I advise you to use back strings as the initial and end of the tag to highlight the word. Instead of h1, it will be `H1`. use  ` back strings around the words for this 👍